packages/dcos-integration-test/extra/test_dcos_log.py

```diff
@@ -0,0 +1,100 @@
+import json
+import requests
+
+RANGE_ENDPOINT = '/system/v1/logs/v1/range/'
+STREAM_ENDPOINT = '/system/v1/logs/v1/stream/'
```

Comment:
Rather than having to add these everywhere, it would nice to add a new client:
in test_util/cluster_api.py
```
@property
def logs(self):
    return self.get_client('/system/v1/logs')
```
and then this test looks like
```
def test_log_text(cluster):
    for node in cluster.masters + cluster.all_slaves:
        response = cluster.logs.get('v1/range/?limit=10', node=node)
        assert response.ok
```